makefile

```diff
@@ -13,8 +13,8 @@ endef
 # because CMake can only handle one toolchain at a time, so we build the MBED-
 # targeted code separately.
 define cmake_build_target_fw
-	mkdir -p build/firmware
-	cd build/firmware && cmake -Wno-dev --target $1 $2 ../.. && make $1 $(MAKE_FLAGS)
```

Comment:
alright. I moved it since the build files looked kinda awkward being double nested in a firmware directory, but didn't think about the tests. I think we'll be able to live with this getting reverted to how it used to be.